packages/esm-patient-search-app/src/compact-patient-search/compact-patient-search.component.tsx

```diff
@@ -95,6 +118,24 @@ const CompactPatientSearchComponent: React.FC<CompactPatientSearchProps> = ({
     }
   }, [focusedResult, bannerContainerRef, handleFocusToInput]);
 
+  useEffect(() => {
+    if (fetchError) {
```

Comment:
I've moved these here so that the resource file doesn't have to deal with any rendering concerns. A post-merge commit should change the resource file extension from `.tsx` to `.ts`.